proto/src/determined/api/v1/experiment.proto

```diff
@@ -226,3 +227,20 @@ message GetExperimentValidationHistoryResponse {
   repeated determined.experiment.v1.ValidationHistoryEntry validation_history =
       1;
 }
+
+// Request to create a new experiment.
+message CreateExperimentRequest {
+  // Experiment context.
+  repeated determined.util.v1.File model_definition = 1;
+  // Experiment config.
+  google.protobuf.Struct config = 2;
```

Comment:
I think this needs to be a string. The experiment config is most frequently submitted in YAML. We could use `google.protobuf.Struct` and have the client (cli, sdk, whatever) muck with it and convert it to this protobuf-struct form but I think it's better to just accept it and leave it alone, for audit purposes (people will want to see exactly what was submitted) and for simplicity (then we can just call yaml.Unmarshal and get the same ser/deser/error messages we're used to).